python/nav/portadmin/handlers.py

```diff
@@ -278,6 +290,24 @@ def is_configurable(self) -> bool:
             return False
         return True
 
+    def get_poe_state_options(self) -> Tuple[PoeState, ...]:
+        """Returns the available options for enabling/disabling PoE on this netbox"""
+        raise NotImplementedError
```

Comment:
Meant to be called by the frontend to populate a dropdown menu or something. the user can then select one of the PoeState options, and this would then be fed into `set_poe_state`

Reply:
Does the return value very specifically need to be a tuple?  Usually I prefer annotating things as `Sequence` unless there is a expectation of being able to use the result value specifically as a tuple or list object.

There is the same thing with `dict` vs. `Mapping`, but I must admit I haven't been using the latter as much as I probably should have.

Reply:
Tuple was used to signify that this is not something that is meant to be manipulated in any way, since it just informs the static choices. Havent used Sequence myself but I can try it out here :)